diff --git a/third_party/spdlog/include/spdlog/fmt/bundled/format.h b/third_party/spdlog/include/spdlog/fmt/bundled/format.h
index 4111333..d7d08cf 100644
--- a/third_party/spdlog/include/spdlog/fmt/bundled/format.h
+++ b/third_party/spdlog/include/spdlog/fmt/bundled/format.h
@@ -485,20 +485,11 @@ inline auto get_data(Container& c) -> typename Container::value_type* {
   return c.data();
 }
 
-#if defined(_SECURE_SCL) && _SECURE_SCL
-// Make a checked iterator to avoid MSVC warnings.
-template <typename T> using checked_ptr = stdext::checked_array_iterator<T*>;
-template <typename T>
-constexpr auto make_checked(T* p, size_t size) -> checked_ptr<T> {
-  return {p, size};
-}
-#else
 template <typename T> using checked_ptr = T*;
 template <typename T> constexpr auto make_checked(T* p, size_t) -> T* {
   return p;
 }
-#endif
 
 // Attempts to reserve space for n extra characters in the output range.
 // Returns a pointer to the reserved range or a reference to it.
 template <typename Container, FMT_ENABLE_IF(is_contiguous<Container>::value)>
